seb_openedx/audit.py

```diff
@@ -0,0 +1,69 @@
+"""Custom audit log for seb_openedx"""
+
+import logging
+
+from django.contrib.admin.models import LogEntry, ADDITION, CHANGE, DELETION
+from django.contrib.contenttypes.models import ContentType
+from django.utils import six
+from django.db.utils import ProgrammingError
+from seb_openedx.constants import SEB_NOT_TABLES_FOUND
+from seb_openedx.models import SebCourseConfiguration
+
+LOG = logging.getLogger(__name__)
+
+
+def get_action(config, instance):
+    """Get the action by simulating the normal behavior of the function to_django_model."""
+    if config:
+        if instance:
+            return CHANGE
+        return ADDITION
+    return DELETION
+
+
+def get_change_message(data):
+    "Get the change message to be displayed in the admin."
+    data = {key.lower(): value for key, value in data.items()}  # keys to lowercase
+    message = ', '.join(data.keys())
+    return 'Changed {}'.format(message)
```

Comment:
@felipemontoya if we need we can do something like that
`Changed from the API {}'.format(message)`
and in ModelAdmin we can use the method `log_change` and change the message to 'Changed from the admin'